lib/board.rb

```diff
@@ -34,4 +34,8 @@ def moves_remaining?
     available_squares = @squares.count { |square| square.is_a? Integer }
     available_squares > 0
   end
+
+  def clear_squares
```

Comment:
This works, but why not just create a new empty board?

Reply:
Good point. Looks like I was overthinking this!